### DISCUSSION TITLE
Negotiated prices array

### DISCUSSION THREAD

**Discussion by Mauvila** [CMSgov/price-transparency-guide#549] [Q&A] [answered]
In the In Network JSON schema, specifically in the "Negotiated Rate Details" object, there is an array of "Negotiated Prices".  What is the purpose of having multiple values for this?  I would assume that it would be for different negotiated_type values, billing_class values, etc.  But I was looking at a BCBS Alabama In Network JSON file and it had a negotiated_prices array that looked like this:

> [
> 	{
> 		"negotiated_type": "negotiated",
> 		"negotiated_rate": 2276.0,
> 		"expiration_date": "2022-06-30",
> 		"service_code": [
> 			"21",
> 			"22",
> 			"24"
> 		],
> 		"billing_class": "professional"
> 	},
> 	{
> 		"negotiated_type": "negotiated",
> 		"negotiated_rate": 2706.16,
> 		"expiration_date": "2022-06-30",
> 		"service_code": [
> 			"21",
> 			"22",
> 			"24"
> 		],
> 		"billing_class": "professional"
> 	},
> 	{
> 		"negotiated_type": "negotiated",
> 		"negotiated_rate": 2942.87,
> 		"expiration_date": "2022-06-30",
> 		"service_code": [
> 			"21",
> 			"22",
> 			"24"
> 		],
> 		"billing_class": "professional"
> 	},

It is basically an array of almost identical negotiated_price objects, differing only by the negotiated_rate value.  
If BCBS Alabama messed up, then I'm ok--maybe a validation tool needs to look for this type of thing.  But otherwise, I'm not clear why these various negotiated_price objects exist.  (Maybe these are all old prices?)

**Comment by shaselton-usds** [comment, marked as answer]
@Mauvila The `negotiated_prices` is an array of [`Negotiated Price Objects`](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#negotiated-price-object) because there could exist multiple prices for an item/service for the same group of provider groups that would dependent upon such things as service codes, modifiers, negotiation types (for capitation -- there might exist a "derived" and a "negotiated" prices), etc -- the point of the array was to avoid having to duplicate listing provider networks multiple times for multiple rates.
